source/hardware/controllers/akai_mpk_mini_2.rst

```diff
@@ -0,0 +1,23 @@
+Akai MPK Mini mkII
```

Comment:
Can you rename the file to `akai_mpk_mini_mkii.rst`?

Reply:
sure